frontend/src/components/Input/index.js

```diff
@@ -0,0 +1,43 @@
+import { PropTypes } from "prop-types";
+import styles from "./index.module.css";
+
+const Input = ({ inputType, disabled, error }) => {
+  let texts = {
+    text: { placeholder: "Ariana", errorMessage: "Enter your name." },
+    email: {
+      placeholder: "arina@fortunemail.com",
+      errorMessage: "Invalid email.",
+    },
+  };
+
+  return (
+    <>
+      {error ? (
+        <div className={styles.container}>
+          <span>{texts[inputType].errorMessage}</span>
+          <input
+            disabled={disabled}
+            type={inputType}
+            placeholder={texts[inputType].placeholder}
+            className={`${styles.input} ${styles.inputError}`}
+          />
+        </div>
+      ) : (
+        <input
+          disabled={disabled}
+          type={inputType}
+          placeholder={texts[inputType].placeholder}
+          className={styles.input}
+        />
```

Comment:
I would prefer to see the happy path first.

Reply:
Done